packages/react-centra-checkout/src/types/global.d.ts

```diff
@@ -6,3 +6,7 @@ interface Window {
     reInitiate: (plugin: string) => void
   }
 }
+
+interface GlobalEventHandlersEventMap {
+  centra_checkout_callback: CustomEvent<Record<string, unknown> | undefined>
+}
```

Comment:
question: The only thing I'm wondering if this would need to be exported for when a consumer of this package subscribes to our custom events. Do you perhaps know?

Reply:
It could.
Though, i wonder if it would be easier to apply the types on a per-project basis, if one want to manually listen for `centra_checkout_callback` events.